src/codegen/TypeScriptEngine.ts

```diff
@@ -33,12 +43,30 @@ export default class TypeScriptEngine {
         )
     }
 
-    // TODO: create class method.
+    public createClass(classDecl: ClassDecl): ClassDeclaration {
+        // Add function declarations
+        let classMembers: ClassElement[] =
+            classDecl.functions.map((func: FuncDecl) => this.createMethod(func));
```

Comment:
Is `classDecl.functions` ever going to be `undefined` or `null`?

Reply:
Nope currently we make a empty array in parse regardless of whether there are functions or not in the program.

Reply:
Yeet!